data/remnant/remnant missions.txt

```diff
@@ -875,10 +879,63 @@ mission "Remnant: Technology Available"
+			`	"Of course, maintaining the secrecy of our capabilities is also very important - and the reason why you had to prove yourself before we let you purchase our ships and outfits - but it is secondary to the fundemental secrecy of our existence. Ideally, no-one should know we exist. Those who do know, should ideally think of us as little more than refugees struggling to survive in a hostile environment`
+			label feast
+			`	He waves dismisively. "I am sure you have more questions, but let us leave it there for now. Our meals await, and, well, I need to show you some stuff in regards to our security procedures. So finish your meal, and let's head over to <destination>."`
+				accept
+	on enter
+		conversation
+			`Once you reach space, the prefect interrupts your normal routine. "I'm sure you noticed that your 'license' isn't exactly a normal ID card. There's a reason for that, namely that it has a whole bunch of extra circuitry in it, and includes a couple sensors on the surface. I don't understand everything in it, but the security specialists assure me that it is incredibly resistance to being hacked or compromised."`
+			choice
+				`	"Okay, sounds good."`
+					goto continue
+				`	"How do you know what regular licenses look like?"`
+			`	The old man smiles. "While we might not have access to modern licenses, we do have all our old entertainment programs from before the exile. Some of them included details such as what licenses looked like. While things might have changed in the intervening years, licenses were historically governed by petty bureaucracy, and those are very resistant to change. So it seemed very likely that things have not changed much. And definitely not so much as to resemble ours." He seems amused, but then his smile fades.`
+			`	"Then again, ancestral space never designed their licenses to withstand much. They are incentivized to not be so secure, are they not? Enough money and the right connections, and licenses for anything can be acquired. These" he holds up a Remnant license "not so much." He shakes his head and becomes serious. "Now, your license has a handful of connection pads on the bottom, which can be connected to your ship's IFF. When you are inbound for landing, simply tap this code on the license, and the cube will verify the pattern as well as your fingerprints, a few other biometrics to ensure you are indeed living and concious, and then transmit an encrypted code. Our IFF systems will verify the transmitted signal and respond accordingly."`
+			choice
+				`	"That sounds thorough."`
+					goto thorough
+				`	"So why haven't I been shot down yet?"`
```

Comment:
Oo, I like that you tie it back to the bit about the guns pointing at the player earlier. Nicely done.

Reply:
Thank you.